### PR TITLE
Provide images, scripts, and docs for running Django admin commands

### PR DESCRIPTION
**Pull request by jeancochrane**
# Overview

In the process of developing #5, I discovered that it was not obvious how to create new migrations after editing the Grout data model. This PR creates a new Docker image and provides a `scripts/django-admin` script that can be used to run arbitrary [`django-admin` commands](https://docs.djangoproject.com/en/dev/ref/django-admin/), including `makemigrations`, which will automatically generate new migrations.

The PR also updates the README to include a section documenting how to run migrations, and adds a new test to make sure that the config file used by `django-admin` is kept up to date.

## Notes

- After reading a [StackOverflow thread discussing different ways of supporting migrations in Django apps](https://stackoverflow.com/questions/37528286/how-to-make-migrations-for-a-reusable-django-app) (recommended by @ddohler) I decided that it made the most sense for Grout to have a migrations settings file in the repo, as opposed to an entirely separate project devoted to running migrations. This seemed to me like the least obtrusive solution. If you disagree, let me know!

## Testing

- Run `./scripts/update` to build new containers, and `./scripts/django-admin makemigrations` to try the new migration function.
- Optionally, run `./scripts/test config` to make sure that the sample settings file, `grout/settings_migrations.py`, works properly.
    - Since the settings file will have to get loaded when you run `./scripts/django-admin makemigrations`, this step will not really test anything new. However, `./scripts/test config` is now part of the test suite by default (in order to make sure that future changes to the Grout settings file configuration are always reflected in the `settings_migrations.py` file), so it's worth running it separately to confirm that it works.

### CODE REVIEW

File: grout/settings_migrations.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+For more information on this file, see
+https://docs.djangoproject.com/en/1.11/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.11/ref/settings/
+"""
+
+# You won't deploy this app to production alone, so don't worry about the
+# secret key.
+SECRET_KEY = 'super duper secret'
```
Comment (flibbertigibbet) [inline grout/settings_migrations.py]: :speak_no_evil:

**Review by flibbertigibbet — APPROVED** [review]
:+1: 

Might consider moving the `migrations` command to a generalized Django command script.

File: grout/settings_migrations.py (hunk 1 above)
Comment (jeancochrane) [reply grout/settings_migrations.py]: Haha! Is this OK to leave in? My understanding is that `SECRET_KEY` is used for creating hashes for things like auth tokens and session cookies, so it doesn't matter in the context of generating migrations.

Comment (flibbertigibbet) [reply grout/settings_migrations.py]: yes, this should be fine for development environments. I'm just amused that this is a more super duper default string than the one we usually use. :smile:

Comment (jeancochrane) [reply grout/settings_migrations.py]: Credit where credit's due: it's an old habit I picked up from @evz, whose code marginalia is legendary.

**Comment by evz** [comment]
Ask me to name a variable some day. There are more shades of subtlety between “stuff” and “junk” than most people care to acknowledge.

**Comment by jeancochrane** [comment]
Great call on the generalized Django command script @flibbertigibbet! I did a quick refactor to organize things around `./scripts/django-admin`, which should provide more flexibility. I've updated the PR title and lead comment to reflect the new setup. Would you like to take another look, or do you think it's good to go?

**Comment by flibbertigibbet** [comment]
Looks good!

**Pull request merged**